src/adagrams.js

```diff
@@ -1,8 +1,104 @@
 const Adagrams = {
+      //WAVE ONE: Our first job is to build a hand of 10 letters. To do so, add a function called drawLetters inside of the Adagrams object in src/adagrams.js. 
   drawLetters() {
-    // Implement this method for wave 1
+    const alphabet = {
```

Comment:
I really like that you store the letter frequencies and then build the pool on the fly - this is much easier to read than a giant array full of 9 'A's, 2 'B's, etc.